packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -62,7 +62,7 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
   /** Optional selection model to track selection state.  */
   selection?: ISelection;
 
-  /** Controls how/if the details list manages selection. */
+  /** Controls how/if the details list manages selection. Options include none, single, multiple */
```

Comment:
A dream solution would automatically pull in the options from `SelectionMode` in case they change in the future. But documenting them like this is definitely an improvement over leaving it a mystery.

Reply:
Agreed.  We were looking into the JSDoc options, but none of them parsed as expected so this is a short term fix so the answer is available. I may leave the issue open till I figure that part out.